55800 Error entering order: ** OE Release Master already exists with …

tyndmar opened the pull request:
…r-no 37521. (132)

File changed: oe/oe-rel.a
Added loop if seqence returns a previously used r-no
This (1) acquires a valid r-no for the release and (2) updates the sequence so next number will be properly incremented

Commit:
55800 Error entering order: ** OE Release Master already exists with r-no 37521. (132)
File changed: oe/oe-rel.a
Added loop if seqence returns a previously used r-no
This (1) acquires a valid r-no for the release and (2) updates the sequence so next number will be properly incremented

brad-advantzware reviewed (changes requested):
move uniqueness validation inside the procedure

Commit:
55800 Error entering order: ** OE Release Master already exists with r-no 37521. (132)
Files changed:
oe/oe-rel.a - moved logic to check for out-of-sequence to oe/getNextRelNo.p
oe/getNextRelNo.p - see above
